src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabDetails/components/InstanceTypeDescription.tsx

```diff
@@ -25,7 +26,8 @@ const InstanceTypeDescription: FC<InstanceTypeDescriptionProps> = ({ vm }) => {
   const None = <MutedTextSpan text={t('None')} />;
 
   const itMatcher = getInstanceTypeMatcher(vm);
-
+  const itKind = getInstanceTypeModelFromMatcher(itMatcher);
```

Comment:
can you rename this const to itModel? would make this more clear

Reply:
sure